local/README.md

```diff
@@ -1,3 +1,55 @@
+# Building The Alpha SDK
+
+## Required tools
+* git
+* cargo
+* docker
+
+## Clone Bottlerocket
```

Comment:
Should this be in a different commit? It doesn't seem related to removing Release.toml.